processing/src/main/java/io/druid/query/filter/LongValueMatcherColumnSelectorStrategy.java

```diff
@@ -28,7 +28,7 @@
   @Override
   public ValueMatcher makeValueMatcher(final LongColumnSelector selector, final String value)
   {
-    final Long matchVal = DimensionHandlerUtils.convertObjectToLong(value);
+    Long matchVal = DimensionHandlerUtils.convertObjectToLong(value);
```

Comment:
reverted to final, got dropped in an intermediate change and I forgot to put it back